python/tvm/relay/frontend/tensorflow.py

```diff
@@ -2896,15 +2903,29 @@ def _parse_import_prerequisites(self, graph):
         """
         missing_operators = set()
         for node in graph.node:
+            try:
+                from tensorflow.python.framework import op_def_registry
```

Comment:
Can you confirm is op_def_registry is not part of all TF versions ? pls confirn.
If this is not in 1.x we shouldn't error as the front end is compatible to TF 1.x now.

Reply:
@srkreddy1238 ,
Thank you Siva. op_def_registry is part of both 1.x and 2.x versions. So there would not be an exception in this case.

Reply:
Hence we don't need this try. We could join it with all tf imports above :)

Reply:
yes makes sense. Thanks Siva :)